src/components/EditAccountModal.tsx

```diff
@@ -346,7 +346,7 @@ function EditAccountModal({
               errors={errors}
               isSubmitted={isSubmitted}
             />
-            <Text color="yellow" my={4} fontSize="xs" textAlign="center">
+            <Text color="orange" my={4} fontSize="xs" textAlign="center">
               Change the parameters below on your own risk.
               <br />
               Changing any of them will change the account&apos;s address.
```

Comment:
I know it's not the part of this PR but I noticed the repetitions "change, changing, change...", so I wanted to suggest  a rephrased warning:
```suggestion
              Any changes to the parameters below will alter the account address.
              <br />
              Proceed at your own risk.
```

Reply:
Agree here. Otherwise looks good.